test/controllers/dashboard_admin_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class DashboardAdminControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
Code that you do not use, delete it.

Reply:
Just delete the code?, I mean, can leave the file in blank? @YilKanda

Reply:
I am not referring to that, what I am saying is that the code that is not necessary and that for this reason is _commented_, it is better to eliminate it.

Reply:
Ok, got it

Reply:
Done.